src/components/button-group/button-group.tsx

```diff
@@ -102,7 +99,8 @@ export class ButtonGroup {
                 <span role="gridcell">
                     <input
                         type="radio"
-                        name={this.radioGroupName}
+                        tabindex="0"
+                        name={button.id}
```

Comment:
`name` is the _group_ that the radio-button belongs to. It should be the same for each radio-button in a group, because that's how the browser knows which radio-buttons belong together. Only one radio-button in each group can be selected at any one time.

The way radio-buttons are controlled with the keyboard is that tab focuses the currently selected button. If the user wishes to change the currently selected radio-button, they can do that using the arrow-keys. See docs here for more info: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio

If this is not the behaviour we want, we shouldn't be using radio-buttons. I think this _is_ the behaviour we want though, because it means that a user that doesn't wish to change the selected button can skip past the whole set with two tab-presses, regardless of the number of items in the set.

Reply:
Aha. Now I understand. Thanks for the clarification @adrianschmidt Then we can drop the first commit in this case. 

I can't answer whether we want this behaviour or not. I guess it's very hard to guess for keyboard users (but perhaps not for blind users). Not sure :/